server/events/command/project_context.go

```diff
@@ -96,7 +96,12 @@ type ProjectContext struct {
 // SetScope sets the scope of the stats object field. Note: we deliberately set this on the value
 // instead of a pointer since we want scopes to mirror our function stack
 func (p ProjectContext) SetScope(scope string) {
-	p.Scope = p.Scope.SubScope(scope) //nolint
+	p.Scope = p.Scope.SubScope(scope).Tagged(map[string]string{
+		"workspace":         p.Workspace,
+		"terraform_version": p.TerraformVersion.String(),
+		"project":           p.ProjectName,
+		"project_path":      p.RepoRelDir,
```

Comment:
Can we also add the repo?

Reply:
Full ProjectContext

https://github.com/runatlantis/atlantis/blob/569031ab2369c1bc4f5e89761d81c94b9420b528/server/events/command/project_context.go#L20

`BaseRepo` would be nice to add
Maybe `Pull` for the PR number